easybuild/easyconfigs/s/sentinelsat/sentinelsat-1.2.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'sentinelsat'
+version = '1.2.1'
+
+homepage = 'https://github.com/sentinelsat/sentinelsat'
+description = """
+Sentinelsat makes searching, downloading and retrieving the metadata of Sentinel 
+satellite images from the Copernicus Open Access Hub easy.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [('binutils', '2.38')]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('tqdm', '4.64.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('html2text', '2020.1.16', {
+        'checksums': ['e296318e16b059ddb97f7a8a1d6a5c1d7af4544049a01e261731d2d5cc277bbb'],
+    }),
+    ('geojson', '3.0.1', {
+        'checksums': ['ff3d75acab60b1e66504a11f7ea12c104bad32ff3c410a807788663b966dee4a'],
+    }),
+    ('geomet', '1.0.0', {
+        'checksums': ['0020a4426469934fb58f541cdc23f27c0c0fbbb3d003ee2cb76bb7ffa96a7506'],
+    }),
+    (name, version, {
+        'checksums': ['d38de0ae1e167d7311426973b0207380e9fa1c4a6f5353dc58bd83bf88b5db4d'],
+    }),
+]
+
+sanity_pip_check = True
+
```

Comment:
@WilleBell would you consider adding a `sanity_check_commands`, perhaps `sentinelsat --version`?